_projects/2023/100509696/100509696.Rmd

```diff
@@ -0,0 +1,170 @@
+---
+title: "New dataviz project"
+description: |
+  A short description of the post.
+author: JIEUN PARK
+date: 2023-12-11
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=TRUE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+```
+
+## Packages and libraries
+
+```{r}
+#install.packages("readxl")
+library(readxl)
+#install.packages("reshape2")
+library(reshape2)
+#install.packages("dplyr")
+library(dplyr)
+library(tidyr)
+library(tidyverse)
+library(scales)
+#install.packages("ggcharts")
+library(ggplot2)
+library(ggalluvial)
+#install.packages("ggsankey")
+library(ggsankey)
+
+```
+
+This graph explains how the electricity generation sources have been changed in the United States from 2001 to 2017 over 16 years. The data covers all 50 states from the United States. Sources are arranged in order of percentages and it has crossover features in the graph. 
+
+Usually, over 16 years, coal, nuclear, and natural gas are the main resources of electric generation in the U.S. According to the article, the ranking of different sources have not been changed very much almost over 17 years for 13 states. In this other 37 states, both the percentages of electricity generated and the relative ordering for different sources have changed noticeably. Therefore, the crossovers of different sources for the electiricty generation in the graph are affected mostly by these 37 states. 
+
+## Original graph
+```{r}
+knitr::include_graphics("~/Desktop/data visualization/ElectricitySourcesLN-superJumbo.webp")
+```
+
+## Data and Data Processing
+
+```{r}
+data <- read_excel("~/Desktop/data visualization/Net_generation_United_States_all_sectors_annual.xls")
```

Comment:
The same for the data. All datasets used here should be put in this directory and pushed as part of the PR.